.github/workflows/buildAndPublish.yml

```diff
@@ -44,13 +44,13 @@ jobs:
             fi
             sha=${{github.sha}}
             short_commit_hash=${sha:0:7}
-            version_tag=v${{steps.extract_version.outputs.version}}-$branch_tag-${{github.run_number}}-$short_commit_hash
+            version_tag=${{steps.extract_version.outputs.version}}-$branch_tag-${{github.run_number}}-$short_commit_hash
           fi
           
           git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com" && git config --global user.name "$GITHUB_ACTOR"
           
           # Tag the commit with the correct version
-          git tag -a $version_tag -m "Tagging $version_tag"
+          git tag -a v$version_tag -m "Tagging $version_tag"
```

Comment:
In services we don't put this v, in the SDK we do. Do we want to keep it for the cli?

Reply:
No strong opinion from my side. Keep it like it is for now and create a ticket to align?